R/cache.R

```diff
@@ -173,7 +185,7 @@
 setGeneric("Cache", signature = "...",
            function(FUN, ..., notOlderThan = NULL,
                     objects = NULL, outputObjects = NULL, algo = "xxhash32",
-                    cacheRepo = NULL) {
+                    cacheRepo = NULL, sideEffect= FALSE, makeCopy = FALSE, quick = FALSE) {
```

Comment:
these are simply parameters that can be passed via the `...`; these don't need to be in the method signature (as this complicates the method definitions).